package.json

```diff
@@ -79,7 +79,8 @@
     "preact-cli": "^3.0.0-rc.3",
     "preact-render-spy": "^1.2.1",
     "prettier": "^1.18.2",
-    "puppeteer": "^1.19.0"
+    "puppeteer": "^1.19.0",
+    "size-plugin": "github:kuldeepkeshwar/size-plugin#develop"
```

Comment:
I'd rather depend on a fixed npm package here. Just pulling the got repository is dangerous for as in case something bad happens (thinking of `left-pad` here).

Reply:
yup, let's hold it until I move changes from `kuldeepkeshwar/size-plugin#develop` to `https://github.com/GoogleChromeLabs/size-plugin`

Reply:
for ref: https://github.com/GoogleChromeLabs/size-plugin/pull/21